.github/CODEOWNERS

```diff
@@ -3,7 +3,6 @@
 # be requested for review when someone opens a pull request.
 
 *                             @eriknordmark
-/api/                         @deitch
 /build-tools/                 @deitch
 /libs/depgraph/               @milan-zededa
```

Comment:
Do you plan to remove references to /libs/* from here and add them to CODEOWNERS of eve-libs in a separate PR?

Reply:
Yes, that was a slip-up. Fixing now.

Reply:
https://github.com/lf-edge/eve-libs/pull/3

and

https://github.com/lf-edge/eve-api/pull/5